src/Adapters/EventSource.Net46/EventSourceListeningRequest.cs

```diff
@@ -0,0 +1,72 @@
+﻿// -----------------------------------------------------------------------
```

Comment:
It is actually a Microsoft requirement to have them.
We should add them everywhere that they are missing.